examples/usecases/transformers-next-item-prediction-with-pretrained-embeddings.ipynb

```diff
@@ -0,0 +1,5658 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    wf = nvt.Workflow(out + 'description_vector')
</pre><p>Can you combine a list with a string?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>good question! 🙂 don't believe so, just tried that a second ago and Python does the sane thing (instead of converting the string to an array of chars or something like that) and throws an error</p><p> </p><p>here however out is not a <code>list</code>  but a <code>&lt;Node Categorify&gt;</code> so it all works </p>